api/commands/run.ts

```diff
@@ -264,7 +270,9 @@ export class Runner {
       taskResult.status = this.cancelled
         ? dataform.TaskResult.ExecutionStatus.CANCELLED
         : dataform.TaskResult.ExecutionStatus.FAILED;
-      taskResult.errorMessage = e.message;
+      const warehouse = this.graph.projectConfig.warehouse;
+      const capitalisedWarehouse = `${warehouse.charAt(0).toUpperCase()}${warehouse.slice(1)}`;
```

Comment:
let's not bother, this will end up being super weird for e.g. "Bigquery" and "Sqldatawarehouse". i'm fine with just lowercasing it, but if you prefer then feel free to add a user visible name mapping along the lines of the stuff we have near `WarehouseType`